apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -263,7 +259,10 @@ const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReading
                 const { data: { dv, sender, organization, readings } } = args
 
                 if (readings.length > READINGS_LIMIT) {
-                    throw new GQLError(ERRORS.TOO_MUCH_READINGS(readings.length), context)
+                    throw new GQLError({
+                        ...ERRORS.TOO_MUCH_READINGS,
```

Comment:
```suggestion
                        ...ERRORS.TOO_MANY_READINGS,
```

Reply:
..and in other places where it's used

Reply:
too much to do it in all places in this pr